skytemple_files/patch/handler/team_stats_pain.py

```diff
@@ -0,0 +1,84 @@
+#  Copyright 2020-2024 Capypara and the SkyTemple Contributors
+#
+#  This file is part of SkyTemple.
+#
+#  SkyTemple is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  SkyTemple is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with SkyTemple.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import (
+    annotations,
+)  # i have no clue what this does but every other patch i looked at had it (i looked at 2 patches)
+
+from typing import Callable
+
+from ndspy.rom import NintendoDSRom
+
+from skytemple_files.common.util import read_u32, get_binary_from_rom
+from skytemple_files.common.ppmdu_config.data import (
+    Pmd2Data,
+    GAME_VERSION_EOS,
+    GAME_REGION_US,
+    GAME_REGION_EU,
+    GAME_REGION_JP,
+)
+from skytemple_files.patch.category import PatchCategory
+from skytemple_files.patch.handler.abstract import AbstractPatchHandler, DependantPatch
+
+ORIGINAL_INSTRUCTION = 0xE3190004  # hex representation of tst r9,0x4 (which we are changing to tst r9,0x5)
+OFFSET_US = 0x22C0DE8 - 0x22BCA80
+OFFSET_EU = 0x22C1728 - 0x22BD3C0  # location of tst r9,0x4 minus starting point of overlay 10
+OFFSET_JP = 0x22C2590 - 0x22BE220
+
+
+class TeamStatsPainPatchHandler(AbstractPatchHandler, DependantPatch):
+    @property
+    def name(self) -> str:
+        return "TeamStatsPain"
+
+    @property
+    def description(self) -> str:
+        return (
+            "If available, change a team member's portrait in the Team Stats menu to pain when in critical condition."
```

Comment:
It'd probably help to make this patch's description translatable by [importing `_`](https://github.com/SkyTemple/skytemple-files/blob/10b7fa7e4f7d23e88fd2ea0bde3af0d53c41179f/skytemple_files/patch/handler/actor_and_level_loader.py#L23) and [using it preceding a string](https://github.com/SkyTemple/skytemple-files/blob/10b7fa7e4f7d23e88fd2ea0bde3af0d53c41179f/skytemple_files/patch/handler/actor_and_level_loader.py#L52)